alpha/scripts/utils/updateKuserRegisterInfoGroupId.php

```diff
@@ -0,0 +1,33 @@
+<?php
+require_once('/opt/kaltura/app/alpha/scripts/bootstrap.php');
+
+if ($argc < 2)
+{
+	die("Usage: php $argv[0] groupId <realrun | dryrun> \n");
+}
+
+$groupId = $argv[1];
+$dryrun = true;
+if($argc == 3 && $argv[2] == 'realrun')
+{
+	$dryrun = false;
+}
+
+$kuserKgroupList = KuserKgroupPeer::retrieveKuserKgroupByKgroupId($groupId);

```

Comment:
@ravitshalem  I talked to Eran. He will provide a dump of all kusers IDs which will be an input to the script. It will be easier since there are many users in this group (currently 18,710 users)